standalone/src/models/product-type/product-type/presets/tshirt.ts

```diff
@@ -0,0 +1,45 @@
+import { type TBuilder } from '@/core';
+import {
+  AttributeDefinitionRest,
+  AttributeDefinitionGraphql,
+  AttributeDefinition,
+} from '../..';
+import {
+  CompatModelBuilder,
+  GraphqlModelBuilder,
+  RestModelBuilder,
+} from '../builders';
+import type {
+  TProductType,
+  TProductTypeGraphql,
+  TProductTypeRest,
+} from '../types';
+
+export const restPreset = (): TBuilder<TProductTypeRest> =>
+  RestModelBuilder()
+    .name('T-shirt Product Type')
+    .attributes([
+      AttributeDefinitionRest.presets.countryOfOrigin().buildRest(),
+      AttributeDefinitionRest.presets.size().buildRest(),
+    ]);
+
+export const graphqlPreset = (): TBuilder<TProductTypeGraphql> =>
+  GraphqlModelBuilder()
+    .name('T-shirt Product Type')
+    .attributeDefinitions({
+      results: [
+        AttributeDefinitionGraphql.presets.countryOfOrigin().buildGraphql(),
+        AttributeDefinitionGraphql.presets.size().buildGraphql(),
```

Comment:
You don't need to use the `buildGraphql` method here. That's only needed for the compatibility model.

```suggestion
        AttributeDefinitionGraphql.presets.countryOfOrigin().build(),
        AttributeDefinitionGraphql.presets.size().build(),
```

Reply:
Fixed